src/identities.rs

```diff
@@ -79,6 +81,22 @@ where
     }
 }
 
+///Implements Zero as the empty string
+#[cfg(feature = "std")]
+impl<'a> Zero for Cow<'a, str> {
+
+    fn is_zero(&self) -> bool {
+        match self {
+            Self::Borrowed(str) => str.len() == 0,
+            Self::Owned(string) => string.len() == 0
+        }
```

Comment:
Doesn't `self.len()` work? I would expect this to automatically `Deref` without needing a manual `match`.

Reply:
Yes, that must've slipped my mind